### PR TITLE
Creating landing page (instead of the old one that was only a picture)

### PR DESCRIPTION
**Pull request by danitLevi**
-Add new images (animated-logo.gif and background.png) into the static folder
-Adding new "get started" button that leads to the explore page (the old one was just a picture )
-remove the old picture that was displayed on the landing page from
static folder

related to #54

### CODE REVIEW

File: home/templates/home/landingpage.html [hunk 1]
```diff
@@ -1,7 +1,44 @@
 {% extends "home/base.html" %}
 {% load static %}
 {% block content %}
+
+<style type="text/css">
+        body{
+         background: url('{% static "home/background.png" %}') no-repeat center center fixed;
+         -webkit-background-size: cover;
+         -moz-background-size: cover;
+         -o-background-size: cover;
```
Comment (liornoy) [inline home/templates/home/landingpage.html:10]: Maybe create a folder in the static called CSS where you can make CSSfile and put all the style there, as suggested by Barak

**Review by liornoy — COMMENTED** [review]
Except for the CSS style in the HTML it looks great

File: home/templates/home/landingpage.html (hunk 1 above)
Comment (danitLevi) [reply home/templates/home/landingpage.html:10]: I added .

**Comment by avivz450** [comment]
I think you should label this PR as status/in-progress

**Comment by danitLevi** [comment]
@ifireball @AviadP
This PR is ready for review

**Comment by ifireball** [comment]
Since this depends on another PR, I cannot review this yet.

**Comment by danitLevi** [comment]
@ifireball @AviadP
This PR is ready for review

**Comment by danitLevi** [comment]
@ifireball @ AviadP
This PR is ready for review

**Comment by rebeccaTubman** [comment]
LGTM

**Review by ifireball — APPROVED** [review]

**Pull request merged**